src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -84,6 +87,19 @@ public function getImpersonatePath(string $identifier): string
         return $this->impersonateUrlGenerator->generateImpersonationPath($identifier);
     }
 
+    public function userIsGranted(UserInterface $user, mixed $attribute, mixed $subject = null, ?string $field = null): bool
```

Comment:
Do we _need_ the field parameter ?

Reply:
IMO it makes sense to keep the same features as `is_granted()`

Reply:
Well it's not documented nor tested if i'm not wrong... 

but above all i believe this was a sort of bridge for Symfony ACL.. that is not the most active project around (correct me if i'm wrong here)

 https://github.com/symfony/acl-bundle/pulls?q=is%3Apr+is%3Aclosed
 
 But maybe while we can it's the good time to reconsider this?

Reply:
I didn't mean to resolve this one without explanation, sorry. Follow up PR welcome on this topic as I tend to agree we don't need this anymore, at least not for new code.